webpack.config.js

```diff
@@ -0,0 +1,75 @@
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
```

Comment:
Cleanup this WP config and double-check that it works properly in browser and node.

Reply:
removed as dead code